Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -583,19 +520,20 @@ public class FeaturevisorInstance {
     //      return getValueByType(variableValue, "json") as T | undefined;
     //    }
     //  }
-}
+    
+    static public func createInstance(options: InstanceOptions) -> FeaturevisorInstance? {
```

Comment:
is it possible to keep the function on a "global" scope of the package?

so that users can import the package and use the function right away:

```swift
import FeaturevisorSDK

let f = createInstance(...)
```

I don't know if having it as a static property is more idiomatic to Swift. my reason for asking this is purely for the sake of maintaining API parity, nothing more.

Reply:
Yes, we can stick to the "global" scope. I'm not a huge fan of it but in this case is rather simple factory method and we will have the same approach as other SDKs as you mentioned so you convinced me :D

Reply:
Done